product-management/src/login/Login.jsx

```diff
@@ -1,5 +1,54 @@
-import React from "react";
+import React, { useState } from 'react';
 
-export const Login = () => {
-  return <div>Login Details</div>;
-};
+function Login() {
+  const [username, setUsername] = useState('');
+  const [password, setPassword] = useState('');
+
+  const handleUsernameChange = (event) => {
+    setUsername(event.target.value);
```

Comment:
pass the event.target.value as a param like below

const handleUsernameChange = (value) => {
    setUsername(evalue);
    }